src/index.js

```diff
@@ -190,6 +190,47 @@ function getMatchingElements (elementIterator, ast, multiple) {
   return results
 }
 
+function getMatchingElementsByTagName (elementIterator, tagName) {
+  const results = []
+  let element
+  while ((element = elementIterator.next())) {
+    if (tagName.toLowerCase() === element.tagName.toLowerCase() || tagName === '*') {
```

Comment:
```suggestion
  const tagNameAsLowerCase = tagName.toLowerCase()
  while ((element = elementIterator.next())) {
    if (tagName === '*' || tagNameAsLowerCase === element.tagName.toLowerCase()) {
```

Micro-optimization, but may be worth it if the DOM is large.